controllers/accounts.go

```diff
@@ -0,0 +1,25 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	ledgerContext "github.com/RealImage/QLedger/context"
+	"github.com/RealImage/QLedger/models"
+	"github.com/julienschmidt/httprouter"
+)
+
+func GetAccountsInfo(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	app := r.Context().Value("app").(*ledgerContext.AppContext)
```

Comment:
@sudhirj Not able to introduce `AppContext` as an argument to the handler, since adding additional arguments changes the function signature which is not compatible to register the handler here`router.GET("/v1/accounts", ContextMiddleware(appContext, controllers.GetAccountsInfo))`

@anuradhag